### ISSUE TITLE
Add "exclude" option

### ISSUE THREAD

**Issue by EvgenyOrekhov** [wapmorgan/PhpCodeFixer#18]
Many projects keep all the sources in the project root directory (here is an example of an official project template for one of the popular PHP frameworks: https://github.com/yiisoft/yii2-app-basic). If one wants to analyse the whole project excluding the `vendor` directory, they have to enumerate all the directories manually:

```sh
phpcf assets/ commands/ config/ controllers/ ...
```

It would be more convenient to be able to exclude some directories from analysis, e. g.

```sh
phpcf --exclude vendor/ ./
```

**Comment by wapmorgan** [comment]
Will be released soon.

**Comment by wapmorgan** [comment]
Released as **2.0.10**.
